agent-transport/src/jvmMain/kotlin/com/epam/drill/agent/transport/http/HttpAgentMessageTransport.kt

```diff
@@ -29,30 +30,36 @@ import org.apache.hc.core5.http.ClassicHttpResponse
 import org.apache.hc.core5.http.ContentType
 import org.apache.hc.core5.http.HttpHeaders
 import org.apache.hc.core5.http.io.entity.ByteArrayEntity
+import org.apache.hc.core5.http.io.entity.EntityUtils
+import org.apache.hc.core5.http.message.BasicHeader
 import org.apache.hc.core5.ssl.SSLContextBuilder
 import mu.KotlinLogging
 import com.epam.drill.agent.transport.AgentMessageTransport
 import com.epam.drill.common.agent.transport.AgentMessageDestination
 
 private const val HEADER_DRILL_INTERNAL = "drill-internal"
-
-private const val API_KEY_HEADER = "X-Api-Key"
+private const val HEADER_API_KEY = "X-Api-Key"
 
 class HttpAgentMessageTransport(
-    adminAddress: String,
-    private val apiKey: String,
+    serverAddress: String,

```

Comment:
renaming `admin` to `server` brings even more confusion
lets keep the `admin` for now

Reply:
thats a transport - it doesn't "need to know" about admin